barista-http/Dockerfile

```diff
@@ -20,12 +20,12 @@ COPY barista-http/src /project/barista-http/src
 # Build and package
 RUN mvn -q -f barista-http/pom.xml package
 
-## Create Image
-FROM adoptopenjdk/openjdk8-openj9:x86_64-ubi-minimal-jre8u232-b09_openj9-0.17.0
-COPY --from=build /project/barista-http/target/lib /project/lib
-COPY --from=build /project/barista-http/target/*-runner.jar /project/app.jar
-WORKDIR /project
+# Start with OL runtime.
+FROM open-liberty
 
-EXPOSE 8082
+# Copy app binaries
 
-CMD ["java", "-Dkafka.bootstrap.servers=kafka:9093", "-jar", "app.jar"]
+COPY --from=build --chown=1001:0 /project/barista-http/target/barista-http-1.0-SNAPSHOT.war /opt/ol/wlp/usr/servers/defaultServer/apps
+COPY --from=build --chown=1001:0 /project/barista-http/src/main/liberty/config/server.xml /opt/ol/wlp/usr/servers/defaultServer
+
+EXPOSE 8082
```

Comment:
I'm guessing we inherit open-liberty's `CMD` here.  Would it be worth a comment to explain what the image will do when run?

Reply:
Done